generic-qemu.toml

```diff
@@ -0,0 +1,4 @@
+# vyos build for gerneric qemu/kvm environments like proxmox. 
+# Includes "qemu-guest-agent" to improve interoperability with different cloud environments and "cloud-init" (see: https://docs.vyos.io/en/latest/automation/cloud-init.html and https://docs.vyos.io/en/latest/automation/cloud-init.html#prepare-cloud-init-files)
+image_format = "qcow2"
+packages=["qemu-guest-agent", "cloud-init"]
```

Comment:
cloud-init by itself without corresponding cloud-init sources and a config that is adjusted for the fact that user/password/etc. will be populated by cloud-init modules isn't really useful. The simple flavor should only include the guest agent.

Reply:
I understand that the cloud-init package doesn't go well with a "qemu" flavor, but I think I might have choosen the wrong name for the flavour. My goal for this flavour is to provide "ready to use" cloud images, to run in proxmox or openstack, where one would expect both qemu-guest-agent and cloud-init to be present. But it's not specifically tied to proxmox or openstack, so how should we name the flavour, would qemu-cloud-init be fine?

Reply:
If we are to move this forward, in the original wording ("add a QEMU flavor"), then a) rename the flavor to `qemu` b) remove cloud-init from it.

cloud-init isn't going to work correctly for VyOS without corresponding data sources and module configs to make it generate VyOS configs. That would be an entirely different PR.